lgc/builder/DescBuilder.cpp

```diff
@@ -91,6 +96,7 @@ Value *DescBuilder::CreateLoadBufferDesc(unsigned descSet, unsigned binding, Val
         desc = UndefValue::get(descTy);
       } else {
         dwordOffset += node->offsetInDwords;
+        dwordOffset += (binding - node->binding) * node->stride;
```

Comment:
I don't think is being tested.  If I understand the test correctly, the resource is not in the root table.  This code is executed only if the the resource is in the root table.  I think another test should be added to run all of the possible paths.